emissions.md

```diff
@@ -140,7 +159,7 @@ SF<sub>6</sub> emissions from electric transformers scale with electricity consu
 Air pollutant emissions such as sulfur dioxide (SO<sub>2</sub>) and nitrogen oxides (NO<sub>x</sub>) are modeled as:
 
 $$
-E_{t}=A_{t}*EF_{t0}*(1-EmCtrl(pcGDP_{t}))
+E_{t}=A_{t}*{EF}_{t0}*(1-EmCtrl(pc{GDP}_{t}))
```

Comment:
This doesn't actually render any differently for me.  Was there something in particular that looked off in the original?